### PR TITLE
Add definition of duration-shorthand

### PR DESCRIPTION
**Pull request by mmazur**
This is the shortest version I could make that at least standardizes the postfixes.

It does not contain the discussion on how implementations can work. I might add that in a separate PR, but that's lower prio.

### CODE REVIEW

**Review by programmer04 — APPROVED** [review]
lgtm

**Review by davex98 — APPROVED** [review]
Looks good! 🚀

File: README.md [hunk 1]
```diff
@@ -72,6 +74,32 @@ spec:
...
+- *w* – weeks
+- *M* – months
+- *Q* – quarters
+- *Y* – years
+
+Examples: `12h`, `4w`, `1M`, `1Q`, `365d`, `1Y`.
+
+This specification does not put requirements on how (or whether) to implement each
+postfix, therefore implementers are free to pick an implementation that best suits
+their environments.
```
Comment (weyert) [inline README.md:98]: Do you  mean the implementor can decide how many days there in a year or week?

**Review by weyert — APPROVED** [review]
Looks good to me, only had one little question

File: README.md (hunk 1 above)
Comment (mmazur) [reply README.md:98]: Yes. A typical week has 168 hours. In countries with daylight savings time there are also weeks that are 167 and 169 hours long. Same with days – most are 24h, some are 23 and 25h. Years are typically 365d except when they're 366d. How do you define a rolling month? Or a rolling quarter? (And maybe you just don't need those in your implementation, so you'll skip having to think about it.)

The idea is that for now we tell implementers to figure something out that best suits them. And if over time the community comes to a conclusion on how to best approach this, we can update the spec with a recommendation on how we think this should be handled.

**Review by kenfinnigan — APPROVED** [review]

**Review by proffalken — APPROVED** [review]
LGTM

**Review by ian-bartholomew — APPROVED** [review]
LGTM too.  Feel free to merge whenever you want

**Pull request merged**